net-libs/libdatachannel/libdatachannel-0.23.2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="C/C++ WebRTC network lib: Data Channels / Media Transport / WebSockets"
+HOMEPAGE="https://github.com/paullouisageneau/libdatachannel"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/paullouisageneau/libdatachannel.git"
+else
+	SRC_URI="https://github.com/paullouisageneau/libdatachannel/archive/refs/tags/v$PV.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0"
+IUSE="+gnutls"
+
+DEPEND="
+	dev-libs/glib:2
+	net-libs/libnice[!gnutls?]
```

Comment:
Specifically here

Reply:
Indeed, this looks suspicious.

What semantics do you actually want here? Describe it to me verbosely and then I'll see what we should do.

Reply:
I am not sure how we usually handle `gnutls` vs `openssl`. Here `libdatachannel` depends both directly on `gnutls/openssl` and also through its `libnice` dependency, so here are my questions:
-  Do we need to force the dependencies to to align on their SSL/TLS backend or can it be mixed, e.g. libdatachannel uses `gnutls` while `libnice` uses `openssl`
  - Do we need to force rebuilding the dependency `libnice` when the flag state changes for `libdatachannel`